MilitiaShipDescriptionAppender.cs

```diff
@@ -60,39 +60,40 @@ public void AddToDescriptionBuffer(GameEntity_Squad RelatedEntityOrNull, GameEnt
                 for (int x = 0; x < (int)CivilianResource.Length; x++)
                 {
                     GameEntityTypeData entityData = GameEntityTypeDataTable.Instance.GetRowByName(militiaData.ShipTypeData[x], false, null);
-                    if (entityData != null)
-                    {
-                        int count = militiaData.GetShipCount(entityData.InternalName);
-                        Buffer.Add("\n" + entityData.DisplayName + ":");
-                        Buffer.StartColor(UnityEngine.Color.green);
-                        Buffer.Add(" " + count + "/" + militiaData.ShipCapacity[x]);
-                        Buffer.EndColor();
-                        Buffer.StartColor(CivilianResourceHexColors.Color[x]);
-                        Buffer.Add(" (" + (CivilianResource)x + ")");
-                        Buffer.EndColor();
 
-                        int cost = 0;
-                        if (RelatedEntityOrNull.TypeData.GetHasTag("BuildsProtectors"))
-                            cost = (int)(12000 * SpecialFaction_SKCivilianIndustry.CostIntensityModifier(RelatedEntityOrNull.PlanetFaction.Faction));
-                        else
-                        {
-                            double countCostModifier = 1.0 + (1.0 - ((militiaData.ShipCapacity[x] - count + 1.0) / militiaData.ShipCapacity[x]));
-                            int baseCost = entityData.CostForAIToPurchase;
-                            cost = (int)(SpecialFaction_SKCivilianIndustry.CostIntensityModifier(RelatedEntityOrNull.PlanetFaction.Faction) * (baseCost * countCostModifier * (militiaData.CostMultiplier / 100.0)));
-                        }
+                    if (entityData == null)
```

Comment:
Just for clarity.

There is no logic change here. I only removed indentation by doing continue inside this if.

Code is moved more to the left so it's easier to read it and we are able to see more of the code as some lines have long code in them.